### DISCUSSION TITLE
Can't change EF Core tools version  (6.0.8 -> 7.0.5)

### DISCUSSION THREAD

**Discussion by RyT-13** [JetBrains/rider-efcore#165] [Q&A] [answered]
Hello! I need help...
I try to create Migration with Rider EF Core tools. And I get message: "The Entity Framework tools version '6.0.8' is older than that of the runtime '7.0.5'. Update the tools for the latest features and bug fixes."

My configs:
- Project
![image](https://user-images.githubusercontent.com/57671083/236236285-aea4df1c-f94d-487f-a6ce-6432b9087633.png)

- NuGet packages
![image](https://user-images.githubusercontent.com/57671083/236235950-3c8544c1-4562-4537-a75d-d125535d610c.png)

Everywhere version 7.*, but only EF Core tools have version "Global, 6.0.8"...

![image](https://user-images.githubusercontent.com/57671083/236237025-d7473812-d61f-47dc-b4aa-a85a45df2ed8.png)

thanks in advance :)

**Comment by seclerp** [comment, marked as answer]
Hi @RyT-13, EF Core Tools, also called just `dotnet ef` is a [.NET Tool](https://learn.microsoft.com/en-us/dotnet/core/tools/global-tools) executable for design time manipulation of your migrations, DbContexts, etc.

`Microsoft.EntityFrameworkCore.Design` and `Microsoft.EntityFrameworkCore.Tools` (they are almost identical so you can remove one of them) NuGets are a "host" part of the tools that should be available in your startup project and it's used as an entry point for `dotnet ef`.

So you should update your `dotnet ef` - to do so:

- If you installed your tools globally (your case according to the dialog information): `dotnet tool update --global dotnet-ef --version 7.0.5`
- If you installed your tools locally (per solution): `dotnet tool update dotnet-ef --version 7.0.5`

**Reply by RyT-13** [reply]
oh, the mistake was really in this...
thank you very much! :)
